pkg/xen-tools/Dockerfile.in

```diff
@@ -10,6 +10,8 @@ COPY initrd/init-initrd initrd/mount_disk.sh initrd/udhcpc_script.sh /
 COPY initrd/poweroff /sbin/poweroff
 COPY initrd/chroot2.c /tmp/
 COPY initrd/00000080 /etc/acpi/PWRF/
+COPY initrd/ssh_config /etc/ssh/
```

Comment:
Why are you adding ssh to the initrd? (well I'm pretty sure I know the answer which is debugging -- but that doesn't belong in this PR -- that belongs in your private repo).

Reply:
Removed.